Makefile

```diff
@@ -15,7 +15,6 @@ publish:
 		-Psigning.secretKeyRingFile="$(NEXUS_GPG_SECRING_FILE)" \
 		-Psigning.password="$(NEXUS_GPG_PASSWORD)" \
 		-Psigning.keyId="$(NEXUS_GPG_KEY_ID)"
-	@echo "Now you go https://s01.oss.sonatype.org, close the temporarly created staging repository and release it"
```

Comment:
this was technically incorrect, we use "closeAndReleaseSonatypeStagingRepository" which afaik does that for us